PocketKit/Sources/Sync/Operations/SearchList.swift

```diff
@@ -24,19 +24,32 @@ class SearchList: SyncOperation {
         self.events = events
     }
 
-    func execute() async -> SyncOperationResult {
+    override func start() {
+        guard !isCancelled else {
+            finishOperation()
+            return
+        }
+
+        Task {
+            isFinished = false
+            isExecuting = true
```

Comment:
kind of mimicked `FetchArchivePagesOperation`, but after reading this article `https://www.avanderlee.com/swift/asynchronous-operations/`, I think we should update `isFinished` and `isExecuting` here